src/flag_gems/utils/libentry.py

```diff
@@ -14,47 +16,110 @@ def __init__(
         while not isinstance(fn, triton.runtime.JITFunction):
             fn = fn.fn
         self.jit_function: triton.runtime.JITFunction = fn
-        self.kernel_arg_indices = []
+        self.spec_indices = []
+        self.dns_indices = []
```

Comment:
let's be more precise? do_not_specialize_indices

Reply:
done